app/uk/gov/hmrc/gform/summary/SummaryRenderingService.scala

```diff
@@ -438,4 +669,208 @@ object SummaryRenderingService {
 
     renderHtmls(sections, fields)
   }
+
+  def summaryForNotificationPdf(
```

Comment:
Why is `summaryForNotificationPdf` method needed? Why it differs from `summaryForRender`?